RIAssigner/compute/Kovats.py

```diff
@@ -0,0 +1,72 @@
+from typing import List, Iterable
+from RIAssigner.data.Data import Data
+from .ComputationMethod import ComputationMethod
+
+
+class Kovats(ComputationMethod):
+    """ Class to compute the Kovats retention index. """
+
+    def compute(self, query: Data, reference: Data) -> List[float]:
+        """ Compute non-isothermal Kovats retention index.
+        For details see https://webbook.nist.gov/chemistry/gc-ri/.
+
+        Parameters

```

Comment:
Missing parameter `reference` in doc string.

Reply:
Added documentation of compute function